index.html

```diff
@@ -1,277 +1,334 @@
+            </p>
+            <button class="about-button" type="button" autofocus>Get My Resume</button>
+          </article>
+          <hr>
+          <ul class="skills-list">
+            <li class="skill-1">Language
+              <ul class="Languages">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>CSS</li>
+              </ul>
+            </li>
+            <li class="skill-2">Frameworks
+              <ul class="frameworks">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </li>
+            <li class="skill-3">Skills
+              <ul class="skills">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- About me section end -->
 
-     <section id="about" class="about-section">
-      <div class="about-content">
-        <article class="about-me">
-          <h2>About me</h2>
-          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of
-            my
-            work and
-            experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      <!-- contact form section  -->
+      <section id="form">
+        <div class="form-container">
+          <p>
+            I'm always interested <br> in hearing about new projects,<br>
+            so if you'd like to chat <br> please get in touch.
           </p>
-          <button type="button" class="about-button" autofocus>Get My Resume</button>
-          <div class="about-img"></div>
-        </article>
-        <hr>
-        <ul class="skills-list">
-          <li class="skill-img-1"></li>
-          <li class="skill-1">Language
-            <ul class="Languages">
-              <li class="lan-li">Javascript</li>
-              <li>Ruby</li>
-              <li>Html</li>
-              <li>CSS</li>
-            </ul>
-          </li>
-          <li class="skill-img-2"></li>
-          <li class="skill-2">Frameworks
-            <ul class="frameworks">
-              <li class="frame-li">Bootstrap</li>
-              <li>Ruby on Rails</li>
-              <li>RSPec</li>
-              <li>CapyBara</li>
-              <li>Selenium</li>
-            </ul>
-          </li>
-          <li class="skill-img-3"></li>
-          <li class="skill-3">Skills
-            <ul class="skills">
-              <li class="skill-li">Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
+          <form class="form" action="https://formspree.io/f/xdovandv" method="POST">
```

Comment:
The Figma design seems to ask you to acquire all the space for form and form container text.

Kindly take a look.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221350596-867471c9-151a-4c68-86c3-cbbc1a5f55be.png)| ![image](https://user-images.githubusercontent.com/61361037/221350569-78418b63-1d69-4b14-abc0-d62443e3b812.png)|